fuzz/ctap2_commands_parameters_corpus.json

```diff
@@ -0,0 +1,152 @@
+[
+    {
+        "hex": "1903e8",
+        "cbor": "unsigned(1000)",
+        "description": "cbor value"
+    },
+    {
+        "hex": "3829",
+        "cbor": "negative(41)",
+        "description": "cbor value"
+    },
+    {
+        "hex": "c349010000000000000000",
+        "cbor": "-18446744073709551617",
+        "description": "cbor value"
+    },
+    {
+        "hex": "f90000",
+        "cbor": "primitive(0)",
+        "description": "cbor value"
+    },
+    {
+        "hex": "f90001",
+        "cbor": "primitive(1) = 5.960464477539063e-8",
+        "description": "cbor value"
+    },
+    {
+        "hex": "fa7fc00000",
+        "cbor": "primitive(2143289344) = NaN",
+        "description": "cbor value"
+    },
+    {
+        "hex": "f818",
+        "cbor": "simple(24)",
+        "description": "cbor value"
+    },
+    {
+        "hex": "d74401020304",
+        "cbor": "tag 23(h'01020304')",
+        "description": "cbor value"
+    },
+    {
+        "hex": "6449455446",
+        "cbor": "IETF",
+        "description": "cbor value"
+    },
+    {
+        "hex": "62225c",
+        "cbor": "\"\\",
+        "description": "cbor value"
+    },
+    {
+        "hex": "41a8",
+        "cbor": "bytes(a8)",
+        "description": "cbor value"
+    },
+    {
+        "hex": "623a41",
+        "cbor": "text(:A)",
+        "description": "cbor value"
+    },
+    {
+        "hex": "83019f0203ff820405",
+        "cbor": "array [1, [2, 3], [4, 5]]",
+        "description": "cbor value"
+    },
+    {
+        "hex": "9f018202039f0405ffff",
+        "cbor": "indefinite length array [1, [2, 3], [4, 5]]",
+        "description": "cbor value"
+    },
+    {
+        "hex": "5f44aabbccdd43eeff99ff",
+        "cbor": "indefinite byte string (_ h'AABBCCDD', h'EEFF99')",
+        "description": "cbor value"
+    },
+    {
+        "hex": "7f657374726561646d696e67ff",
+        "cbor": "indefinite byte string (_ \"strea\", \"ming\")",
+        "description": "cbor value"
+    },
+    {
+        "hex": "a26161016162820203",
+        "cbor": "map {\"a\": 1, \"b\": [2, 3]}",
+        "description": "cbor value"
+    },
+    {
+        "hex": "bf6346756ef563416d7421ff",
+        "cbor": "indefinite length map {\"Fun\": true, \"Amt\": -2}",
+        "description": "cbor value"
+    },
+    {
+        "hex": "",
```

Comment:
Why is the hex empty when the CBOR is not?

Reply:
Ups my bad. Fixed.